metagraph/CMakeLists.txt

```
@@ -364,10 +337,13 @@ add_library(metagraph-cli STATIC ${metagraph_cli_files})
 add_executable(metagraph "src/main.cpp")
 set_target_properties(metagraph PROPERTIES OUTPUT_NAME "metagraph_${CMAKE_DBG_ALPHABET}")
 
+find_package(BZip2 REQUIRED)
+
 set(METALIBS ${METALIBS}
   -lKMC
-  -lhts -lz -lbz2
+  -lhts -lz
   -lsdsl
+  ${BZIP2_LIBRARIES}
```

It's a dependency of HTSlib, right?
```suggestion
  -lhts -lz ${BZIP2_LIBRARIES}
  -lsdsl
```

---

I didn't notice that, actually. I added this because it's a requirement for KMC.

---

Then `-lKMC ${BZIP2_LIBRARIES}`... to make it clear why we include that.